content/members/127labs/127labs.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" height="42" viewBox="0 0 69 42" width="69" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><clipPath id="a"><path d="m0 0h68.7005v41.156h-68.7005z"/></clipPath><g clip-path="url(#a)"><path clip-rule="evenodd" d="m11.6031 32.0792v-31.851688h-11.6031v4.777758h6.23383v27.07393zm18.5195-32.0792c-4.0952 0-8.1449 1.72909-11.6031 4.55024l2.1386 4.14072c2.6846-2.45713 5.9153-4.00421 8.691-4.00421 3.3216 0 5.3237 1.54708 5.3237 4.09522 0 1.82013-1.547 4.41373-3.8677 6.77983l-11.7396 12.6042v3.9132h22.6602v-4.8233h-15.5618l8.2814-8.7819c3.3672-3.6402 5.8243-7.2349 5.8243-10.46558 0-4.86875-4.1407-8.00842-10.147-8.00842zm38.495 4.32273v-4.095218h-23.1607v9.282488h4.7323l.0455-4.45923h12.3766l-12.3766 27.02843h5.6878zm-21.6028 36.12507v-5.5935h-.7171v6.2748h3.4691v-.6813zm7.9601-.9053.6992 1.5866h.7798l-2.8057-6.2748h-.744l-2.8147 6.2748h.7619l.6992-1.5866zm-.3048-.6903h-2.8147l1.3984-3.1822zm2.7699-3.9979v6.2748h2.6892c1.4163 0 2.2589-.6364 2.2589-1.7032 0-.8605-.5289-1.4521-1.4342-1.5866.735-.1524 1.1563-.6813 1.1563-1.4163 0-.9861-.7619-1.5687-2.0258-1.5687zm.7171.6902h1.8914c.8426 0 1.3356.3586 1.3356.995 0 .6544-.493 1.0219-1.3356 1.0219h-1.8914zm0 2.6982h1.8914c1.0219 0 1.6135.4034 1.6135 1.0846 0 .7172-.5916 1.1205-1.6135 1.1205h-1.8914zm7.5656-3.4153c-1.2997 0-2.1693.6364-2.1693 1.6225 0 2.25 3.5319 1.3715 3.5229 3.0209 0 .6006-.5647.9681-1.4701.9681-.7082 0-1.506-.3227-2.0886-.8875l-.3227.6455c.5916.5647 1.488.9591 2.4023.9591 1.3446 0 2.2679-.6723 2.2679-1.7211.009-2.2769-3.5228-1.4342-3.5228-3.0478 0-.5288.5289-.8336 1.3356-.8336.502 0 1.1653.1524 1.7301.5289l.3048-.6723c-.52-.3496-1.264-.5827-1.9901-.5827z" fill="#643ddb" fill-rule="evenodd"/></g></svg>
```

Comment:
Did you use some automation to generate members directory ? Can you please share script so we can use it in future?